src/parking/parking-lane.ts

```diff
@@ -129,23 +130,27 @@ function wayIsMajor(tags: OsmTags) {
     return tags.highway.search(majorHighwayRegex) >= 0
 }
 
-function getConditions(side: 'left' | 'right', tags: OsmTags): ConditionsInterface {
-    const conditions: ConditionsInterface = { intervals: [], default: null }
-
-    conditions.intervals = parseConditionsByOldScheme(side, tags)
-    conditions.default = parseDefaultCondition(side, tags, conditions.intervals.length)
+function getConditions(side: 'left' | 'right', tags: OsmTags): ParkingConditions {
+    const conditions: ParkingConditions = { conditionalValues: [], default: null }
 
+    conditions.conditionalValues = parseConditionsByNewScheme(side, tags)
+    if (conditions.conditionalValues.length > 0) {
```

Comment:
If I understand this correctly, it's attempting to parse by the new schema, then if no relevent tags are found it parses by the old schema.

Rather than trying the old than new schema, could we write a function to convert all old schema tags to new ones (and unit test this heavily to ensure it is correct), then parse by the new schema?

This would also close https://github.com/zlant/parking-lanes/issues/93. I added a proposal for this method here: https://github.com/zlant/parking-lanes/issues/93#issuecomment-1066608415

Reply:
I think there should be a separate independent method that checks for old tags and suggests migration

Reply:
Which is called during the opening of the edit panel

Reply:
Why suggest migration rather than do it automatically?

- If the user declines the migration we need to parse by the old schema, and support editing using the old schema
- Why should we encourage users to edit to an old schema?

What if we displayed a little banner or "toast" message if the tags are (automatically) migrated? (and the save button would be highlighted too).

Reply:
If it's out of scope in this PR I'm happy to draft a PR for what I'm thinking afterwards :)

Reply:
Now many parking lots are mapped according to the old scheme and they will not be migrated manually soon.
If the user wants to make minor edits, rather than deal with the new scheme, he should leave this option.